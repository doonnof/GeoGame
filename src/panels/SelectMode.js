import React from "react";
import {
  Card,
  CardGrid,
  Div,
  PanelHeader,
  Text,
  Title,
  Panel,
} from "@vkontakte/vkui";

import { IconCountry } from "../components/IconCountry";
import { IconRegion } from "../components/IconRegion";

function SelectMode({ id, goToPanel2, selectMapModel }) {
  return (
    <Panel id={id}>
      <PanelHeader>GeoGame</PanelHeader>

      <CardGrid size="l" spaced>
        <Card mode="shadow" style={{ textAlign: "center" }}>
          <CardGrid size="l" spaced>
            <Title level="1" style={{ marginBottom: 16 }}>
              Выберите режим
            </Title>
            <Card
              onClick={() => {
                selectMapModel.mapType = "001";
                selectMapModel.mapZoom = 1;
                goToPanel2();
              }}
            >
              <Div
                style={{
                  display: "flex",
                  flexDirection: "column",
                  alignItems: "center",
                  gap: "6px",
                  paddingBottom: "20px",
                }}
              >
                <IconCountry />
                <Title level="2">Страны</Title>
                <Text
                  weight="3"
                  style={{ color: "var(--vkui--color_icon_medium)" }}
                >
                  Вам предстоит угадывать страны по всей планете!
                </Text>
              </Div>
            </Card>
            <Card
              onClick={() => {
                selectMapModel.mapType = "RU";
                selectMapModel.mapZoom = 2;
                goToPanel2();
              }}
            >
              <Div
                style={{
                  display: "flex",
                  flexDirection: "column",
                  alignItems: "center",
                  gap: "6px",
                  paddingBottom: "20px",
                }}
              >
                <IconRegion />
                <Title level="2">Россия</Title>
                <Text
                  weight="3"
                  style={{ color: "var(--vkui--color_icon_medium)" }}
                >
                  Попробуйте угадать все регионы России!
                </Text>
              </Div>
            </Card>
          </CardGrid>
        </Card>
      </CardGrid>
    </Panel>
  );
}
export default SelectMode;
