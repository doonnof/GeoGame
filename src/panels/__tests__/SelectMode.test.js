import React from "react";
import renderer from "react-test-renderer";
import SelectMode from "../SelectMode";

it("SelectMode should render correctly", () => {
  const tree = renderer.create(<SelectMode />).toJSON();
  expect(tree).toMatchSnapshot();
});