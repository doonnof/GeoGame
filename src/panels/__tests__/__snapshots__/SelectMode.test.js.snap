// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectMode should render correctly 1`] = `
<div
  className="vkuiPanel vkuiPanel--sizeX-none"
>
  <div
    className="vkuiPanel__in vkuiInternalPanel__in"
    data-tooltip-container="true"
    onClickCapture={[Function]}
    onDragStart={[Function]}
  >
    <div
      className="vkuiPanel__in-before"
    />
    <div
      className="vkuiPanelHeader vkuiInternalPanelHeader vkuiPanelHeader--android vkuiInternalPanelHeader--android vkuiPanelHeader--vis vkuiInternalPanelHeader--vis vkuiPanelHeader--sep vkuiInternalPanelHeader--sep vkuiPanelHeader--vkapps vkuiPanelHeader--no-before vkuiPanelHeader--no-after vkuiPanelHeader--fixed vkuiPanelHeader--sizeX-none"
    >
      <div
        className="vkuiFixedLayout vkuiFixedLayout--vertical-top vkuiPanelHeader__fixed vkuiInternalPanelHeader__fixed"
        data-tooltip-container="fixed"
        style={
          Object {
            "width": undefined,
          }
        }
      >
        <div
          className="vkuiPanelHeader__in"
          data-tooltip-container="fixed"
        >
          <div
            className="vkuiPanelHeader__before vkuiInternalPanelHeader__before"
          />
          <div
            className="vkuiPanelHeader__content"
          >
            <span
              className="vkuiPanelHeader__content-in"
            >
              GeoGame
            </span>
          </div>
          <div
            className="vkuiPanelHeader__after vkuiInternalPanelHeader__after"
          />
        </div>
      </div>
      <div
        className="vkuiSeparator vkuiSeparator--padded vkui-sizeX--compact-mq"
      >
        <hr
          className="vkuiSeparator__in"
        />
      </div>
      <div
        className="vkui-sizeX--regular-mq vkuiSpacing"
        style={
          Object {
            "height": 16,
            "padding": "8px 0",
          }
        }
      />
    </div>
    <div
      className="vkuiCardGrid vkuiInternalCardGrid vkuiCardGrid--spaced vkuiInternalCardGrid--size-l vkuiCardGrid--sizeX-none"
    >
      <div
        className="vkuiCard vkuiCard--mode-shadow"
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <div
          className="vkuiCardGrid vkuiInternalCardGrid vkuiCardGrid--spaced vkuiInternalCardGrid--size-l vkuiCardGrid--sizeX-none"
        >
          <h1
            className="vkuiTitle--level-1 vkuiTypography--normalize"
            style={
              Object {
                "marginBottom": 16,
              }
            }
          >
            Выберите режим
          </h1>
          <div
            className="vkuiCard"
            onClick={[Function]}
          >
            <div
              className="vkuiDiv"
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "flexDirection": "column",
                  "gap": "6px",
                  "paddingBottom": "20px",
                }
              }
            >
              <svg
                fill="none"
                height="64"
                viewBox="0 0 64 64"
                width="64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#8A6DC7"
                  height="64"
                  rx="32"
                  width="64"
                />
                <path
                  d="M44.6162 16.5519L44.0227 15.9565L41.9029 17.9553C40.5462 16.7222 38.9354 15.7438 37.2394 15.0635C35.4166 14.3827 33.5512 14 31.5587 14C22.995 14 16 21.0174 16 29.6081C16 33.138 17.187 36.3702 19.1372 39.007L17.0175 40.9635L17.5262 41.6013C20.7906 45.4715 25.5088 47.7679 30.5113 48.0658V49.299H25.8478C25.3815 49.299 25 49.6817 25 50.1495C25 50.6173 25.3815 51 25.8478 51H36.4464C36.9127 51 37.2942 50.6173 37.2942 50.1495C37.2942 49.6817 36.9127 49.299 36.4464 49.299H32.2069V48.1082C42.0848 47.8103 50 39.6449 50 29.6506C50 24.6747 48.0926 20.0392 44.6162 16.5519ZM31.5587 15.7014C33.0001 15.7014 34.4414 15.9141 35.7558 16.3392C35.7558 16.4243 35.7136 16.5092 35.671 16.5519C35.1623 17.6151 35.8404 18.2105 36.095 18.4656C36.2222 18.5932 36.3494 18.6783 36.4342 18.8483C36.5191 18.9759 36.6885 19.3161 35.8407 20.1666C34.8232 21.1874 34.2722 22.846 35.0352 23.6542C35.2472 23.952 35.8407 25.4404 35.9678 26.1208C35.6709 26.2908 35.2895 26.1208 34.3568 25.4403C34.1873 25.3127 34.0176 25.1852 33.8481 25.1C33.3818 24.8022 33.0426 24.2919 32.746 23.7817C32.3645 23.1863 32.0253 22.676 31.4741 22.3783C30.5836 21.9528 29.7784 22.6761 28.8457 23.5691C28.2522 24.122 27.3617 24.9725 26.8954 24.9725C26.3442 24.9725 26.0476 24.4196 25.7084 23.6966C25.5389 23.3987 25.4115 23.1012 25.242 22.8885C24.9028 22.5482 24.0973 21.4425 23.9701 21.0599C23.9278 20.8472 23.843 20.6344 23.6309 20.4645C23.4189 20.2945 23.1646 20.0818 21.2992 20.3369C23.758 17.4875 27.4464 15.7014 31.5587 15.7014ZM19.7731 22.3358C20.7907 22.1658 21.8929 21.9955 22.4441 21.9531C22.9104 22.8888 23.9703 24.0796 23.9703 24.0796C24.0125 24.122 24.0974 24.2923 24.1397 24.4623C24.5212 25.2276 25.1996 26.6738 26.8529 26.6738C27.9976 26.6738 29.0575 25.6958 29.9902 24.8025C30.2445 24.5474 30.626 24.2071 30.838 24.0371C30.9651 24.2071 31.1349 24.4625 31.3043 24.6749C31.6858 25.2703 32.1521 25.9933 32.9154 26.5035C33.0848 26.5887 33.2123 26.7162 33.3817 26.8438C33.9752 27.2692 34.8657 27.907 35.7984 27.907C36.307 27.907 36.8582 27.6943 37.4094 27.1841C38.2572 26.3336 36.8159 23.3566 36.3496 22.6761C36.3496 22.4634 36.5616 21.9107 37.0279 21.4429C38.1727 20.2945 38.4693 19.1889 37.9606 18.1681C37.7486 17.7427 37.452 17.4451 37.24 17.2748C37.2822 17.1897 37.2822 17.1048 37.3248 17.0197C42.1154 19.2313 45.4221 24.0796 45.4221 29.6508C45.4221 31.7772 44.9558 33.7761 44.108 35.5623L42.0306 34.9669C41.8185 34.9245 41.6065 34.9245 41.4371 35.0094L40.0803 35.6899H38.4269L37.5365 34.4991L36.3069 33.2656C36.0949 33.0529 35.7983 32.9677 35.5014 33.0529L33.9752 33.4356C33.7208 33.478 33.5511 33.6483 33.424 33.861C33.3817 33.9461 33.2968 34.0737 33.2545 34.1588C32.9576 34.1164 32.4916 33.9888 32.0252 33.9037L31.0502 32.6278C30.8808 32.415 30.6262 32.2875 30.3719 32.2875C30.2024 32.2875 29.2271 32.3299 28.6336 32.9253C28.5913 32.9677 28.5487 33.0104 28.5065 33.0529H28.4642H27.3194C26.8531 33.0529 26.4716 33.4356 26.4716 33.9034V35.8171C26.4716 37.0503 27.9556 37.0503 28.4642 37.0503C28.6336 37.0503 29.312 37.2203 29.9481 37.3481C30.0753 37.4757 30.2024 37.646 30.2873 37.7735C30.245 37.816 30.245 37.9011 30.2024 37.9435C30.0753 38.1562 29.9481 38.4538 29.9055 38.794C29.7361 38.7516 29.4392 38.624 29.2272 38.5389C29.1849 38.5389 29.1423 38.4965 29.1 38.4965L27.9127 38.0289C27.6583 37.9438 27.3614 37.9865 27.1497 38.1565C26.9377 38.3265 26.8105 38.5819 26.8105 38.837V39.6024C26.8105 39.6448 26.7682 39.7724 26.7682 39.8575C26.6834 40.1554 26.5988 40.5381 26.7259 40.8783C26.6837 40.9207 26.6411 40.9635 26.5988 41.0059H26.5565H26.175C25.7087 41.0059 25.3272 41.3886 25.3272 41.8564V41.9839C20.8335 39.6875 17.7386 35.0092 17.7386 29.608C17.6955 26.9289 18.459 24.4623 19.7731 22.3358ZM28.9727 40.4957C28.8455 40.2406 28.6758 40.0703 28.4641 39.9854H28.5063C28.8032 40.1554 29.4813 40.4108 29.9903 40.4108C30.8381 40.4108 31.6013 39.6455 31.6013 38.8373C31.6013 38.7949 31.6862 38.6674 31.6862 38.6246C31.8134 38.3695 31.9831 38.0717 31.9831 37.689C31.9831 37.2212 31.6439 36.6258 30.9656 35.9452C30.8384 35.8177 30.7113 35.7752 30.5841 35.7325C30.0755 35.605 28.8885 35.3071 28.4644 35.3071H28.1675V34.7117H28.4644C29.1428 34.7117 29.3122 34.5842 29.8212 34.0739C29.8634 34.0315 29.906 34.0315 29.9906 33.9888L30.8384 35.1371C30.9656 35.3071 31.1353 35.3923 31.3048 35.435C31.8134 35.5625 33.0004 35.8604 33.4245 35.8604C34.018 35.8604 34.4843 35.3926 34.7386 34.9671L35.4593 34.7971L36.2222 35.5625L37.3244 37.0512C37.4939 37.2639 37.7485 37.3914 38.0028 37.3914H40.2919C40.4191 37.3914 40.5462 37.349 40.6734 37.3063L41.9027 36.6685L43.3017 37.0512C40.8428 40.9214 36.5185 43.5157 31.6009 43.5157C29.9898 43.5157 28.4213 43.2178 27.0223 42.7075V42.6651C27.3192 42.6227 27.5309 42.4524 27.8701 42.1121C28.0821 42.0697 28.3788 41.9846 28.6757 41.6867C29.0997 41.2189 29.0999 40.7935 28.9727 40.4957ZM31.5587 46.4075C26.9378 46.4075 22.5288 44.4938 19.3492 41.0914L20.197 40.2833C23.0374 43.303 27.0648 45.2167 31.5163 45.2167C40.0799 45.2167 47.0749 38.1996 47.0749 29.6086C47.0749 25.5684 45.5488 21.9108 43.0476 19.1464L43.8954 18.3383C46.6935 21.4429 48.2197 25.3982 48.2197 29.6085C48.3045 38.8798 40.801 46.4075 31.5587 46.4075Z"
                  fill="white"
                />
                <path
                  d="M20.9886 32.2083C20.8461 32.4934 20.9172 32.814 21.131 33.0278C21.2023 33.0991 21.7012 33.5622 22.2714 33.5622C22.8771 33.5622 23.1977 32.992 23.4828 32.4932C23.5541 32.3507 23.6965 32.1367 23.7323 32.0656C23.9106 31.8874 24.1243 31.6737 24.2667 31.3886C24.4805 30.9253 24.4091 30.4622 24.053 30.1058C23.9105 29.9634 23.5541 29.1437 23.2335 28.3955C23.1622 28.2173 22.9839 28.0391 22.806 27.968L21.8438 27.6474C21.7725 27.6118 21.7014 27.6118 21.6301 27.6118H20.3117C19.9553 27.6118 19.6705 27.8614 19.5992 28.182L19.2785 29.7854C19.2072 30.1418 19.4209 30.5338 19.8129 30.6049C20.098 30.6762 20.4542 30.7831 20.6324 30.89C20.7037 31.0682 20.8819 31.2819 21.2026 31.3888C21.1668 31.7808 21.0244 32.1014 20.9886 32.2083ZM20.8461 29.4646L20.9175 29.0726H21.4877L22.022 29.2508C22.2358 29.7852 22.5564 30.4981 22.8415 30.89C22.806 30.9614 22.7346 30.9969 22.6991 31.0324L22.6278 31.1038C22.5922 30.7118 22.4496 30.2843 21.8439 30.1061C21.737 29.9279 21.4163 29.6786 20.8461 29.4646Z"
                  fill="white"
                />
                <path
                  d="M22.1328 29.7844C22.1622 30.0792 22.3208 30.323 22.3208 30.6305"
                  stroke="white"
                  strokeLinecap="round"
                  strokeWidth="3"
                />
                <path
                  d="M21.6665 20.0677C22.6211 19.8073 23.593 19.5763 24.437 19.0372C24.893 18.7461 25.2781 18.3921 25.7813 18.1774C26.6278 17.8163 27.6028 17.7385 28.4973 17.5633C29.5438 17.3582 30.5851 17.0875 31.6465 16.9696C32.7746 16.8442 33.967 16.794 35.0892 17.0105C35.5405 17.0976 35.3878 18.104 35.3247 18.4265C35.2462 18.8277 35.0956 19.1535 34.9084 19.5115C34.721 19.8699 34.6479 20.2544 34.5228 20.6341C34.2706 21.3996 34.0628 22.0286 34.168 22.8553C34.2474 23.4788 34.3836 24.1471 34.6593 24.7148C34.7198 24.8394 35.0977 25.5223 34.8709 25.0389C34.4085 24.0539 33.3926 23.2576 32.52 22.6505C32.3655 22.5431 32.2979 22.2955 32.2232 22.1353C31.8729 21.3849 31.9768 20.7081 32.5439 20.0881C32.7928 19.816 33.1196 19.6072 33.3423 19.3102C33.4119 19.2174 33.342 19.2141 33.2536 19.2181C32.7042 19.2425 32.1445 19.3577 31.6056 19.4569C30.921 19.5829 30.2511 19.661 29.9986 20.3884C29.8351 20.8592 29.895 21.4967 29.5618 21.8965C28.9454 22.6362 27.7878 22.6823 27.1427 23.408C27.0171 23.5493 26.8991 23.7379 26.6958 23.7697C26.1541 23.8543 25.7866 22.528 25.6107 22.1763C25.4477 21.8502 25.2725 21.6282 25.0239 21.371C24.7178 21.0544 24.8173 20.9233 25.2286 20.7535C26.2754 20.3213 27.4111 20.148 28.3608 19.5149"
                  stroke="white"
                  strokeLinecap="round"
                  strokeWidth="3"
                />
                <path
                  d="M29.0978 34.9917C29.9083 34.9917 30.3635 35.2695 31.0256 35.7526C31.2401 35.9091 31.4307 36.0841 31.65 36.2302C32.0057 36.4674 32.0168 37.1076 32.0594 37.4824C32.0978 37.8203 32.1922 38.1554 32.2436 38.4924C32.3096 38.9243 32.541 39.2634 32.4381 39.7207C32.2498 40.5579 31.1879 40.8565 30.4626 41.0719C29.924 41.2318 29.3853 41.4543 28.8897 41.7167C28.8123 41.7577 28.2674 42.2346 28.2516 42.1978C28.2042 42.0871 29.0389 40.857 29.1422 40.9047C29.5146 41.0766 29.5191 41.7123 29.6505 42.0409C29.8642 42.5749 30.9846 42.5472 31.4248 42.5322C32.4925 42.4957 33.5173 42.2388 34.5945 42.2388C35.9407 42.2388 37.3026 42.1015 38.5695 41.6178C39.2562 41.3556 39.8723 41.0133 40.3983 40.4986C40.6463 40.2559 40.8769 39.9939 41.1353 39.7617C41.5446 39.3937 41.8442 39.0688 42.0463 38.5333C42.1201 38.3377 42.1187 38.0952 42.2783 37.9601C42.3107 37.9327 42.6526 37.6326 42.4864 37.6326C42.26 37.6326 41.8681 37.8779 41.6539 37.9533C40.7966 38.2551 39.9648 38.5235 39.1427 38.9189C38.0915 39.4245 37.1187 39.4165 35.9798 39.2669C35.2383 39.1695 34.6038 38.9126 34.3454 38.1375C34.2094 37.7294 34.0031 36.9875 34.0861 36.5612C34.2604 35.6664 35.527 36.203 35.9286 36.51C36.4195 36.8855 36.6538 37.448 37.1125 37.8475C37.1783 37.9048 37.7078 38.2634 37.6687 38.3081C37.6523 38.3269 37.6187 38.3612 37.5936 38.3696C36.7595 38.6476 35.937 39.0047 35.1575 39.4102C34.74 39.6274 34.3857 39.8219 34.2567 40.2735"
                  stroke="white"
                  strokeLinecap="round"
                  strokeWidth="3"
                />
                <path
                  d="M36.554 36.6577C39.9627 36.7589 42.77 37.0633 44.9846 37.5646C47.2068 38.0677 48.7892 38.7599 49.7882 39.6055C52.5552 41.9479 52.8621 46.1578 50.4348 49.0251C48.0075 51.8924 43.8047 52.2847 41.0377 49.9423C40.0388 49.0967 39.0949 47.6502 38.2319 45.5415C37.3718 43.4401 36.6081 40.7215 35.9457 37.3763C35.9364 37.3295 35.933 37.2811 35.9357 37.2322C35.9539 36.9043 36.2357 36.6483 36.554 36.6577ZM47.6843 46.698C48.7622 45.4247 48.6432 43.5201 47.3789 42.4499C46.1147 41.3796 44.2166 41.5766 43.1386 42.85C42.0607 44.1233 42.1797 46.0279 43.444 47.0981C44.7083 48.1684 46.6064 47.9714 47.6843 46.698Z"
                  fill="white"
                  stroke="#8A6DC7"
                />
              </svg>
              <h2
                className="vkuiTitle--level-2 vkuiTypography--normalize"
              >
                Страны
              </h2>
              <span
                className="vkuiText vkuiText--sizeY-none vkuiTypography--normalize vkuiTypography--weight-3"
                style={
                  Object {
                    "color": "var(--vkui--color_icon_medium)",
                  }
                }
              >
                Вам предстоит угадывать страны по всей планете!
              </span>
            </div>
          </div>
          <div
            className="vkuiCard"
            onClick={[Function]}
          >
            <div
              className="vkuiDiv"
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "flexDirection": "column",
                  "gap": "6px",
                  "paddingBottom": "20px",
                }
              }
            >
              <svg
                fill="none"
                height="64"
                viewBox="0 0 64 64"
                width="64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#3D89FA"
                  height="64"
                  rx="32"
                  width="64"
                />
                <path
                  d="M22.2778 35.6794C26.2816 35.5062 29.9019 33.9895 31.8185 31.6929C31.9888 31.5197 32.2018 31.3894 32.4573 31.3894C32.7128 31.3894 32.9258 31.5194 33.0961 31.6929C35.0555 34.0326 38.6332 35.5062 42.6794 35.6794C42.0831 34.2495 41.2313 32.9494 40.0812 31.8661C38.8886 30.7396 38.7181 28.8762 39.6552 27.403C40.6347 25.843 41.1033 24.0231 41.0183 22.1597C40.7628 17.87 37.2701 14.2741 33.0535 14.0133C30.6684 13.8834 28.411 14.7067 26.6646 16.3531C24.9182 17.9998 23.9812 20.2528 23.9812 22.6797C23.9812 24.3696 24.4497 26.0163 25.3443 27.4027C26.2813 28.8327 26.0683 30.6959 24.8758 31.8227C23.7682 32.9059 22.8739 34.206 22.2776 35.6792L22.2778 35.6794ZM32.5 16.6135C35.7797 16.6135 38.4628 19.3434 38.4628 22.6797C38.4628 26.0164 35.7794 28.746 32.5 28.746C29.2203 28.746 26.5372 26.0161 26.5372 22.6797C26.5372 19.3434 29.2203 16.6135 32.5 16.6135ZM22.7037 45.2124H42.3389C43.1482 43.6089 43.5742 41.8325 43.5742 40.0123C43.5742 39.1022 43.489 38.2357 43.2759 37.4123C41.3593 37.3691 39.5277 37.0656 37.8666 36.5457C35.737 35.8524 33.8628 34.8126 32.5 33.469C31.1369 34.8122 29.263 35.8523 27.1334 36.5457C25.4723 37.0656 23.6407 37.3691 21.7241 37.4123C21.4686 38.4956 21.3833 39.5789 21.4258 40.7054C21.5535 42.2657 21.9796 43.8256 22.7037 45.2124ZM29.0073 38.4524C29.3056 37.4991 30.1147 36.8057 31.0092 36.7625H31.1369C31.6055 36.7625 32.074 36.8924 32.4572 37.1524C32.8833 36.8924 33.3943 36.7189 33.9053 36.7625C34.7999 36.8057 35.6089 37.4991 35.9072 38.4524C36.2479 39.5789 35.8647 40.749 34.7998 41.6155L32.9257 43.0887C32.7554 43.2187 32.5849 43.2619 32.4147 43.2619C32.2444 43.2619 32.0314 43.2187 31.9036 43.0887L30.0723 41.6155C29.0924 40.7057 28.6667 39.5357 29.0072 38.4524H29.0073ZM32.5001 27.0128C34.5872 27.0128 36.3333 25.4961 36.6741 23.4595C34.8852 23.156 33.3944 22.0727 32.5001 20.5995C31.6055 22.1163 30.0721 23.1996 28.3261 23.4595C28.6669 25.4964 30.413 27.0128 32.5001 27.0128ZM33.6074 23.8064C33.6499 23.7632 33.6927 23.7197 33.7352 23.7197C33.7776 23.7197 33.8204 23.6765 33.8629 23.6329C33.9054 23.6329 33.9482 23.5897 34.0332 23.5897C34.3315 23.5465 34.587 23.6329 34.7997 23.8064C34.9705 23.9796 35.0558 24.1964 35.0558 24.4131C35.0558 24.6298 34.9705 24.8465 34.8003 25.0197C34.6297 25.1929 34.4167 25.2796 34.2037 25.2796H34.0334C33.991 25.2796 33.9057 25.2364 33.8632 25.2364C33.8207 25.2364 33.7779 25.1932 33.7354 25.1497C33.693 25.1065 33.6502 25.1065 33.6077 25.0629C33.5652 25.0197 33.5224 24.9762 33.5224 24.933C33.4799 24.8897 33.4799 24.8462 33.4371 24.803C33.3947 24.7598 33.3947 24.7162 33.3947 24.6298V24.4566C33.3519 24.1963 33.4372 23.9796 33.6074 23.8064ZM30.2002 23.8064C30.2427 23.7632 30.2855 23.7197 30.328 23.7197C30.3705 23.7197 30.4133 23.6765 30.4557 23.6329C30.4982 23.6329 30.541 23.5897 30.626 23.5897H30.9668C31.0092 23.5897 31.052 23.6329 31.137 23.6329C31.1795 23.6762 31.2223 23.6762 31.2648 23.7197C31.3072 23.7629 31.35 23.8064 31.3925 23.8064C31.563 23.9796 31.6483 24.1964 31.6483 24.4131C31.6483 24.6298 31.563 24.8465 31.3928 25.0197C31.3503 25.0629 31.3075 25.1064 31.265 25.1064C31.2226 25.1064 31.1798 25.1497 31.1373 25.1932C31.0948 25.1932 31.052 25.2364 30.9671 25.2364H30.7968H30.6266C30.5841 25.2364 30.5413 25.1932 30.4563 25.1932C30.4138 25.1932 30.3711 25.15 30.3286 25.1064C30.2861 25.0632 30.2433 25.0632 30.2008 25.0197C30.0297 24.8462 29.9444 24.6298 29.9444 24.4131C29.9444 24.1964 30.0297 23.9796 30.2002 23.8064ZM28.3686 21.6831C28.7519 20.0799 29.9872 18.8231 31.563 18.4332C31.1798 20.0799 29.9444 21.3365 28.3686 21.6831ZM33.4796 18.4765C35.0555 18.8664 36.2908 20.1232 36.6741 21.7264C35.0558 21.3365 33.8204 20.08 33.4796 18.4765ZM30.6259 38.9724C30.7112 38.6689 30.9667 38.4957 31.1369 38.4957H31.1794C31.4349 38.4957 31.648 38.6257 31.8182 38.8424C31.9884 39.0591 32.2442 39.1891 32.4997 39.1891C32.7553 39.1891 33.0108 39.0591 33.1813 38.8424C33.3091 38.6257 33.5646 38.4957 33.8201 38.4957H33.8625C34.0328 38.4957 34.2886 38.7124 34.3736 38.9724C34.5013 39.4058 34.2883 39.8389 33.8198 40.2288L32.5 41.3121L31.1797 40.2721C30.7112 39.839 30.4982 39.4055 30.6259 38.9724ZM44 47.812V49.5452L21 49.5455V47.8123C21 47.3357 21.3833 46.9458 21.8518 46.9458H43.1481C43.6166 46.9455 43.9998 47.3354 43.9998 47.812L44 47.812Z"
                  fill="white"
                />
                <path
                  d="M24.0316 41.716C20.9486 39.9174 18.5131 38.2304 16.7148 36.6563C14.9109 35.0774 13.7801 33.6396 13.2596 32.3472C11.8171 28.7652 13.5472 24.7021 17.1292 23.2595C20.7112 21.817 24.7743 23.5471 26.2169 27.1291C26.7373 28.4215 26.9186 30.2417 26.7124 32.6301C26.5068 35.0112 25.9202 37.9152 24.9442 41.3485C24.93 41.3987 24.9094 41.4469 24.8831 41.492C24.7098 41.789 24.3286 41.8893 24.0316 41.716ZM18.5743 26.8451C16.9769 27.4884 16.2027 29.3067 16.846 30.9041C17.4893 32.5015 19.3075 33.2758 20.905 32.6324C22.5024 31.9891 23.2766 30.1709 22.6333 28.5735C21.99 26.976 20.1718 26.2018 18.5743 26.8451Z"
                  fill="white"
                  stroke="#2688EB"
                />
              </svg>
              <h2
                className="vkuiTitle--level-2 vkuiTypography--normalize"
              >
                Россия
              </h2>
              <span
                className="vkuiText vkuiText--sizeY-none vkuiTypography--normalize vkuiTypography--weight-3"
                style={
                  Object {
                    "color": "var(--vkui--color_icon_medium)",
                  }
                }
              >
                Попробуйте угадать все регионы России!
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="vkuiPanel__in-after"
    />
  </div>
</div>
`;
